playbooks/roles/jenkins_analytics/defaults/main.yml

```diff
@@ -201,6 +203,9 @@ jenkins_seed_job:
     - id: ANSWER_DISTRIBUTION
       enable: "{{ ANALYTICS_SCHEDULE_ANSWER_DISTRIBUTION | default(true) }}"
       extra_vars: "{{ ANALYTICS_SCHEDULE_ANSWER_DISTRIBUTION_EXTRA_VARS | default('') }}"
+    - id: ANALYTICS_AGGREGATE_TRACKING_LOGS
+      enable: "{{ ANALYTICS_SCHEDULE_AGGREGATE_TRACKING_LOGS | default(true) }}"
+      extra_vars: "{{ ANALYTICS_SCHEDULE_AGGREGATE_TRACKING_LOGS_EXTRA_VARS | default('') }}"
```

Comment:
If you can believe it, there's yet another PR needed, to add a line to [olive-secure jenkins-extra-vars.yml](https://github.com/edx-olive/olive-secure/blob/master/ansible/vars/admin/jenkins-extra-vars.yml#L60):
```yaml
ANALYTICS_SCHEDULE_AGGREGATE_TRACKING_LOGS_EXTRA_VARS: "@{{ ANALYTICS_SCHEDULE_SECURE_REPO_DEST }}/job-configs/aggregate_tracking_logs.yml"
```

This connects up your [aggregate_tracking_logs.yml](https://github.com/edx-olive/analytics-secure/pull/5/files#diff-b89f3ccfd63c4182ed8b5208b9c71b7f) file as the source of the extra vars for this task.